internal/rbd/rbd_journal.go

```diff
@@ -581,9 +598,10 @@ func RegenerateJournal(
 	}
 
 	rbdVol.RequestName = requestName
-	// TODO add Nameprefix also
+	if namePrefix, ok := volumeAttributes["volumeNamePrefix"]; ok {
+		rbdVol.NamePrefix = namePrefix
+	}
```

Comment:
`rbdVol.Owner` is not required to be set for encryption? I don't see it is getting set.